examples/bin/sample_internet_client_platform.rs

```diff
@@ -0,0 +1,55 @@
+use rustls_platform_verifier::Verifier;
```

Comment:
This code is almost exactly the same as `sample_internet_client.rs`. Could you merge them into a single file and have a build-time or runtime option about which root certs to use?